src/test/java/com/ea/restaurant/test/fixture/OrderStatusHistoryFixture.java

```diff
@@ -0,0 +1,73 @@
+package com.ea.restaurant.test.fixture;
+
+import com.ea.restaurant.constants.EtlStatus;
+import com.ea.restaurant.constants.OrderStatus;
+import com.ea.restaurant.constants.Status;
+import com.ea.restaurant.entities.OrderStatusHistory;
+import java.time.Instant;
+import java.util.Optional;
+
+public class OrderStatusHistoryFixture {
+  public static final Long ORDER_ID = 1L;
+  public static final Instant FROM_TIME = Instant.EPOCH;
+  public static final Instant TO_TIME = Instant.EPOCH;
+  public static final OrderStatus FROM_STATUS = OrderStatus.NEW_ORDER;
+  public static final OrderStatus TO_STATUS = OrderStatus.IN_PROCESS;
+  public static final String MONGODB_ORDER_STATUS_HISTORY_UUID = "MONGOUUID";
+  public static final EtlStatus ETL_STATUS = EtlStatus.PROCESSED;
+  public static final Long CREATED_BY = 1L;
+  public static final Instant CREATED_DATE = Instant.EPOCH;
+  public static final Long UPDATED_BY = 1L;
+  public static final Instant UPDATED_DATE = Instant.EPOCH;
+  public static final Status ENTITY_STATUS = Status.ACTIVE;
+
+  public static OrderStatusHistory buildOrderStatusHistory(OrderStatusHistory orderStatusHistory) {
```

Comment:
Rename to `orderStatusHistoryExample`

Reply:
Create a new `OrderStatusHistory` remember this is a fixture and a new object should be built from here. Fiuxtures must avoid side effects, so do not modify an instance, instead create a new one

Reply:
Done.